themes/rebellion/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
no need for this file

we may have to start giving overrides for the license embedded into the package instead